grails-app/controllers/streama/FileController.groovy

```diff
@@ -84,6 +84,17 @@ class FileController {
     respond status: NO_CONTENT
   }
 
+  def removeMultipleFilesFromDisk() {
+    def bulk = params.files
+    def listOfFiles = JSON.parse(bulk)
```

Comment:
There is an easier way, if the params are passed normally, not stringified, then the request looks something like this: 
```
DELETE /file/removeMultipleFilesFromDisk?id=1&id=2&id=4
```
and then in the controller, you can do 
```
def ids = params.list('id')
// ['1', '2', '4']
```
and if you want it not as strings, then you can do 
```
def ids = params.list('id').collect{it.toLong()}
// [1, 2, 4]
```